wagtail/wagtailadmin/static_src/wagtailadmin/scss/core.scss

```diff
@@ -417,19 +417,76 @@ footer,
             display: none;
         }
 
-        img {
-            width: 60px;
+        .wagtail-logo-container {
             float: none;
             margin: auto;
             display: block;
+            width: 60px;
+            height: 75px;
+            position: relative;
+            transition: all 0.25s cubic-bezier(0.28, 0.15, 0, 2.1);
+
+            img {
```

Comment:
We're trying to move away from bare elements in CSS selectors to make Wagtail's CSS more maintainable, as well as using BEM notation. I'd suggest switching those selectors (here and below) to use the following naming:

```html
<div class="wagtail-logo-container">
    <img class="wagtail-logo wagtail-logo__full" />
    <img class="wagtail-logo wagtail-logo__body" />
    <img class="wagtail-logo wagtail-logo__tail" />
    <img class="wagtail-logo wagtail-logo__eyeopen" />
    <img class="wagtail-logo wagtail-logo__eyeclosed" />
</div>
```

Then, each definition can be moved outside of `.wagtail-logo-container` to reduce nesting and CSS specificity.